### PR TITLE
创建了一个clone-fork.md文文件

### PR DESCRIPTION
**Pull request by Mars-david**
Did you want to me join you.

### CODE REVIEW

File: clone-fork.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Clone fork
+---
+**[51CTO](https://www.51cto.com)**
```
Comment (chengxu1120) [inline clone-fork.md:3]: This code must change.

**Review by chengxu1120 — CHANGES_REQUESTED** [review]
must change

File: clone-fork.md (hunk 1 above)
Comment (chengxu1120) [inline clone-fork.md:3]: add a new code

**Review by chengxu1120 — CHANGES_REQUESTED** [review]
add a code

File: clone-fork.md (hunk 1 above)
Comment (Mars-david) [reply clone-fork.md:3]: ok  how to change?

Comment (Mars-david) [reply clone-fork.md:3]: how to add do you want?

**Review by chengxu1120 — APPROVED** [review]

**Pull request merged**